### DISCUSSION TITLE
Possible Bug: Double Notifications On Lock Screen on iOS

### DISCUSSION THREAD

**Discussion by hnt3b453** [balzack/databag#25] [General]
I noticed this morning that when I get a message from a person, I get a double visual notification for that message on my lock screen/in my notification center in iOS.

**Comment by balzack** [comment]
This sounds like a bug...I'll look into it this weekend.

**Reply by hnt3b453** [reply]
thank you!

**Comment by balzack** [comment]
this should be fixed now with an update of the docker container.
